dato = "three-stranded, two-stranded, DNA, hydrogen bonds, sequence-specific, DNA double helix, oligo-{α]-thymidyiate, pbotocrosslinking, oligodeoxyribonucleotides, homopyrimidine-homopurine, double-stranded DNA, triple helix formation, major groove, antisense oligodeoxynucleotide, hybridization site, transcript, mRNA, pre-mRNA, antisense oligonucleotides, splicing, mRNA, pre-mRNA, target gene, sequence-specific, transcription factors, chromatin remodelling, DNA, target sequences, double helix, oligonucleotide, triple helix formation, pyrimidine oligodeoxynucleotides, purine oligodeoxynucleotides, double-stranded DNA, double-stranded DNA, intermolecular triplexes, target sequence, Hoogsteen, reverse Hoogsteen, molecular mechanisms, antigene strategies, replication, transcription, recombination, repair, in vitro, in vivo, nucleic acid, c-myc gene, promoter, binding protein, purine-rich strand, protooncogene, triple-helix formation, homopyrimidine oligonucleotide, homopurine-homopyrimidine sequence, double-stranded DNA, molecular modeling, triple helix, DNA target, third strand, oligonucleotides forming triple helices, antigene oligonucleotide, duplex DNA, purine-rich oligonucleotides, purine motif, triple helical complexes, purine-purine-pyrimidine triple helical complexes, double helical DNA, third strand, triple-helix formation, DNA, RNA, sequence specific DNA, binding protein, DNA damage site, triple-helix formation, third strand, triplex formation, arrested elongation complexes, DNA templates, promoter, third strand, triple-helical structure, transcription, RNA pol II, triple-helical DNA, gene expression, antigene strategy, oligonucleotide, major groove, double-helical DNA, triple helix, triplex-specific, triple-helical structure, duplex target, single-stranded targets, major groove, double-helical DNA, oligopurine-oligopyrimidine sequences, triple-helical complexes, oligonucleotides, polypurine sequence, transcription elongation, promoter, RNA polymerase II, triplex site, DNA mimic, DNA oligomers, RNA oligomers, PNA, mRNA, block protein synthesis, antisense strategy, duplex invasion, arrest transcription, gene sequence, sequence-targeted drug, triplex-DNA, purine, binding site, pyrimidine, base-pair, triplex formation, double-stranded DNA. oligonucleotide, intermolecular GA oligonucleotide, homoduplex, internal GA repeats, homoduplexes, triplexes, nucleic acid, molecular biology, genetic regulation, genetic expression, double-stranded nucleic acids, triple-stranded nucleic acids, Watson-Crick, base pairs, intermolecular, intramolecular, backbone, nucleic acid base, hydrogen bonding (H-bonding), helical chain, duplex, helix formation, single-stranded, RNAs, ribonucleic acids, deoxyribonucleic acid, DNA, thermodynamic principles, computational, duplex, triplex stability, thermodynamics, calorimetry, triple helices, antigene strategies, melting transitions, pH effects, salt effects, ligand-binding effects, triplex motif, oligonucleotide triple-H, triple helix hybridization, nucleic acid, base pairing, DNA, multistrand structures, H-DNA complexes, triple helices, triple helix, thermodynamic, intermolecular purine-purine-pyrimidine triple helix, third strand, antiparallel, duplex, hydrogen bond, single strands, enhance, DNA, triplex formation, duplex formation, in vitro, in vivo, triplexes, promoter, regulatory enhancer sequences, transcription, oligonucleotide, collinear triplex formation, binding site, oligonucleotide, target DNA, triplex, mRNA, c-myc, triple helix formation, oligonucleotides, polypurine tracts, binding site, purine tracts, third strand, triplexes, molecular mechanics, dynamics simulations, triplets, canonical conformations, triplexes, hydrogen bond, DNA, triplex forming oligonucleotides, TFO, major groove, Watson-Crick duplex, Hoogsteen base-pairing, reverse-Hoogsteen base-pairing, DNA helix forming, triple helices, molecular modeling, Fourier transform infrared spectroscopy, vibrational spectroscopy, third-strand, base pairing, anti-glycosidic torsion angle, Watson-Crick duplex, N-type sugars, S-type sugars, parallel strand, homopurine strand, Hoogsteen hydrogen bonding, triplex formation, triple helix, duplex strand, single strand, third strand, purine strand, intermolecular triplexes, pyrimidine strand, thermodynamic, UV melting curves, parallel strand, oligonucleotides, antisense strategy, mRNA, physicochemical studies, triple-helices targeted, polypurine strand, oligodeoxyribonucleotide, antiparallel complex, Fourier transform infra-red spectroscopy, electrophoresis, molecular modeling, reverse Hoogsteen triplets, S-type sugar, anti-gene strategy, oligonucleotide X-ray structures, nucleic acids, atomic positions, intramolecular interactions, intermolecular interactions, RNA, nucleic acid, base pairs, DNA, in vivo, double helix, oligonucleotide duplex, A conformation, B conformation, polynucleotides, RNA, DNA, B-DNA, hydrogen bonding, three-stranded structures, four-stranded structures, purine-rich sequences, DNA triplexes, DNA quadruplexes, H NMR Spectroscopy, in vivo, triplex form, two-dimensional gel electrophoresis studies, homopurine-homopyrimidine duplexes, H-DNA, major groove, Hoogsteen pairing, Watson-Crick duplex, pyrimidine motif, G-quadruplexes, G-quartet structure, three-dimensional structures, X-ray crystallographic studies, DNA aptamers, in vitro, intranucleotide, internucleotide, anti-nucleotides, parallel triplex helix, antiparallel triplex helix, crystal structure, nucleic acid triplexes, sequence-specific interactions, single-stranded polynucleotides, double helix, triplexes, genetic recombination, in vivo, antigene therapy, triplex helix, x-ray crystal structure, oligonucleotide, DNA triplex helix, Fourier Transform Infrared Spectra, triple stranded polynucleotides, homopurine, homopyrimidine, double stranded helixes, sugar conformations, nucleic acids, , S-type sugars, triple helical polynucleotidic structures, triple helix, Watson-Crick base-paired, adenine strands, thymine strands, pseudorotational symmetry, Hoogsteen-paired, polynucleotides, backbone conformations, three strands, triple helix, X-ray fiber diffraction, double helical, B-form DNA, DNA triplex helix, molecular structures, sequence-specific recognition, DNA duplexes, third strands, triplex formation, gene regulation, site-specific, genomic DNA, DNA triplexes, NMR, molecular dynamics computations, pyrimidine-purine-pyrimidine DNA triplex, third pyrimidine strand, major groove, parallel purine strand, antiparallel purine strand, anti-glycosidic torsion angles, Watson-Crick duplex, intramolecular triplex, B-DNA, nuclear magnetic resonance spectroscopy, purine-purine-pyrimidine DNA triple helix, solution structure, glycosidic torsion angles, NMR, R*RY triplexes, A*AT triples, intramolecular purine, reversed Hoogsteen pairing, DNA hydration, DNA triple helix, NMR spectroscopy, nucleic acid structures, gene regulation, Hoogsteen paired strands, binding specific sites, phosphate groups, DNA, triplex helix, molecular modeling, base triplet, crystal structure, oligonucleotide-drug complex, B-form DNA, A-form DNA, DNA triple helices, x-ray diffraction, Fourier transforms, triplex, oligonucleotides, computer models, parallel intramolecular triple helices, antiparallel intramolecular triple helices, UV spectroscopy, south-type sugar conformation, DNA hydration, parallel triple helix, Fourier transforms infrared, molecular dynamic simulations, thymidine strands, hydrated water molecules, pyrimidine-purine-pyrimidine DNA, hairpin triplex, oligonucleotides, Crick-Hoogsteen groove, methyl group, thymidines, Watson-Hoogsteen groove, S-type sugars, N-type sugars, methyl group, triple-helical oligonucleotide, polynucleotide, triplex, oligonucleotide triplex, glycosidic bond, Watson-Hoogsteen groove, phosphodiester backbone, furanose rings, molecular mechanics, Y+ -R- -R+-type DNA triple helix, polypurine strand, homopurine strand, homopurine-homopyrimidine, duplex DNA, glycosidic torsions, third strand, purine strand, anticonformation hydrogen-bonds, Hoogsteen hydrogen-bonds, reverse-Hoogsteen hydrogen-bonding, syn orientation, triplex, electrostatic interactions, in vacuo simulations, DNA triple helices, triplexes, molecular mechanics, thermodynamics, major groove, modeling study, third strand, triplex formation, B-DNA duplex target, A-DNA, phosphate groups, canonical duplex structures, pyrimidine-purine-purine DNA triple helix, molecular dynamics simulation, GC-T triplet, hydrogen bonds, antiparallel structure, non-hydrogen-bonded base pairs, duplex DNA, X-ray crystallography, NMR spectroscopy, biomolecules, DNA, molecules, biomolecular systems, molecular model, molecular system, quantum-mechanical model, classical model, stochastic model, molecular simulation, biomolecular simulation, dielectric models, stochastic dynamic simulation, solvation force models, solvent models, DNA, proteins, lipids, sugar, DNase I, intermolecular purine-purine-pyrimidine triple-helical DNA, oligonucleotides, triplex-forming, duplex DNAs, purine-motif triplex formation, triple-helix formation, DNA triple helices, triplexes, major groove, c-Ki-ras promoter; R*R*Y triple helices, kinetics, activation energy, DNase I, oligonucleotides, bimolecular, intermolecular triple hélices, GA- GT oligonucleotides, triplex formation, conformational dynamics, cross-links, triplex DNA, pyrimidine-purine-pyrimidine motif intramolecular, DNA triple helix, Hoogsteen base-pair, Watson-Crick base pair, triple helix formation, oligonucleotide, pyrimidine-purine-pyrimidine structure, binding affinity, triple-helical complex, oligonucleotides, oligodeoxyribonucleotides, inhibit specific, gene transcription, antigene strategy, formation of triple helices, pyrimidine-motif triplexes, purine-motif triplex formation, phosphorothioate oligonucleotides, phosphodiester oligonucleotides, triplex formation, guanosine-rich oligodeoxyribonucleotide, polypurine DNA target, DNase I, endonuclease, intermolecular purine - purine - pyrimidine triple-helical DNA, G/A-rich oligonucleotide, oligonucleotide, duplex DNA, triplex formation, tetraplex, G/T-rich oligonucleotides, electrophoretic mobility, oligonucleotide ODN, four-stranded, oligonucleotides, tetraplexes, purine-pyrimidine-purine triplex, monovalent cations, metal cations, guanine tetramers, intermolecular interaction, tetramers, binding sites, dehydration enthalpies, nucleic acids, four-stranded, quadruplex structure, non-Watson–Crick hydrogen bonds, monovalent cations, circular dichroism, single strands, sequence-specific, d(G·C)-tract, double-stranded DNA, G*G·C-based triple helical, G-tract single strand, DNase I, triplex, metal cations, triplex target sites, multistranded nucleic acid, triple helix-forming oligonucleotide, phosphodiester, polypurine tract, purine.pyrimidine inversion, TATA box, promoter, human HER2 gene, computer modeling, triple helix, in vivo, ex vivo, Fourier transform infrared spectroscopy, electrophoresis, oligonucleotide, MCF7 cells, breast cancer, mRNA, transcription, sequence specific, DNase I, antiparallel DNA triple helices, DNA, homopurine target sites, GA-rich oligonucleotides, GT-rich oligonucleotides, antiparallel orientation, triplets, triplexes, third strand, GT oligonucleotides, homopurine, pyrimidines, gel retardation, dissociation constant, oligonucleotide association, human HER2, Neu oncogene targeting, triple helix-forming oligonucleotides, Watson- Crick duplexes, triplex, molecular modeling, triple helix structures, triple helices, S-type sugars, molecular mechanics, stereochemical model, triplex, Hoogsteen, state diagram, phase diagram, disproportionation, antisense, antigene, nucleic acid, triple helix forming oligonucleotides, thermodynamic, triplex, pur. pyr sequences, Hoogsteen pairs, chemical probes, DNA, single-stranded DNA, intramolecular triplex, mirror repeats, non-B DNA, copolymers, DNA, H-DNA, DNA molecule, single-stranded polypyrimidines, third strand, negative supercoiling, triple-stranded H-DNA, DNA triplex, H-DNA, gene-drugs, triplex-forming oligonucleotides, PNA, oligonucleotide-directed, triple-helix formation, triple-helical complexes, NMR, molecular dynamics, triple-helical DNA, triple-helix, oligonucleotide, double-helical DNA, antisense, DNA, oligonucleotide, RNA, potential drugs, antisense strategy, sequence-specific, translation, transcription, selective inhibition, antisense oligonucleotides, therapeutic agents, B-DNA, oligonucleotides, triple helices, double-stranded DNA, intermolecular triplexes, binding affinities, Hoogsteen hydrogen bonding, reverse Hoogsteen hydrogen bonding, Watson-Crick hydrogen bonding, triplex-forming oligonucleotides, TFO, antigene strategy, TFOs, replication, transcription, recombination, repair, in vitro, ex vivo, double-stranded nucleic acids, duplexes, adenine, thymine, uracil, guanine, cytosine, Watson-Crick base pairs, intermolecular interactions, hydrogen bonding, H-bonding, single-stranded, intramolecular base-base interactions, furanose-phosphate interactions, ribonucleic acid, RNAs, base-phosphate interactions, deoxyribonucleic acid, DNA, thermodynamic principles, nucleic acids, base stacking, triplexes, molecular mechanics, computational chemistry, X-ray diffraction, nuclear magnetic resonance spectroscopy, molecular dynamics simulations, melting transitions, pH effects, salt effects, ligand-binding effects, thermodynamics, calorimetry, triple helices, antigene strategy, melting transitions, pH effects, salt effects, ligand-binding effects, model oligonucleotide triple-H, drug-binding, nucleic acids, hydratation effects, double-stranded DNA, H-DNA, RNA, site-specific inhibition, transcription, homopurine-homopyrimidine Watson-Crick duplex, major groove, Watson-Crick DNA duplex, third strand, Hoogsteen-type hydrogen bonds, purine-rich oligonucleotides, pyrimide-rich oligonucleotides, reverse Hoogsteen hydrogen bonds, triplex-forming oligonucleotides, triple-helical complexes, triplex, thermodynamic, quadruplexes, Watson-Crick hairpins, intramolecular triplexes, B- DNA, triplex, phase diagram, calorimetry, conformational changes, melting transitions, DNA, oligonucleotide, intramolecular triple helix, antiparallel duplex, Watson-Crick hairpin, Hoogsteen interactions, major groove, circular dichroism, Hoogsteen hairpin, protonation state, scanning calorimetry, thermodynamic characterization, triplex, Hoogsteen strand, coiled conformation, triple helix, hairpin, calorimetry, gel mobility, nuclease digestion, oligonucleotide, coil transition, purine decamer, pyrimidine decamer, intramolecular triple helix, thermodynamic properties, triplex, oligonucleotide, purine-pyrimidine-pyrimidine motif, intramolecular triple helix, Hoogsteen strand, triplex, duplex, third strand, parallel orientation, purine strand, melting temperatura, triple helix, hairpin transition, thymines, cytosines, nucleic acid, base pairs, nucleotides, melting transition, RNA, hairpin, coil, duplex, single strand, thermodynamic, thermodynamic, nucleic acids, bimolecular transitions, triplex DNA, anti-gene strategy, transcription, physiological conditions, triple-stranded, double-stranded, single-stranded DNA, triplex DNA, circular dichroism spectroscopy, ultraviolet spectroscopy, melting transitions, anti-gene therapeutic,  oligonucleotide-directed, triple helix formation, single DNA, pyrimidine-purine-pyrimidine structure, oligonucleotide, triple-stranded DNA, thermodynamic, pyrimidine oligonucleotide, double-helical, hairpin, triple-helix formation, thermodynamics, gel electrophoresis, DNA triple helices, methylated strand, DNA triplexes, triplex, cytosine methylation, methylcytosine, oligonucleotides, Hoogsteen hydrogen bonding, DNA triplex, base pair, duplex, third strand, triplex, intermolecular triplex, single strand, van't Hoff enthalpy, duplexes, triplets, DNA, RNA, thermodynamics, DNA, triplex formation, oligomers, triple-helical, nucleic acids, base-pair, hydrogen bonding, H-DNA triplex, triplexes, gene expression, nucleases, single strand, duplex DNA, antigene, RNA, antisense, therapeutic strategies, thermodynamic parameters, DNA triplexes, duplexes, enthalpy parameters, protonation, salt concentration, pH values, buffer types, triplets, third strand, thermodynamic characterization, oligomeric DNA triplex, triplex, major-groove Hoogsteen, pyrimidine, single strand, Watson-Crick, duplex, purine-rich strand, pyrimidine-rich strand, Watson-Crick base pairs, third-strand oligonucleotides, circular dichroism spectroscopy, ultraviolet spectroscopy, electrostatic effects, DNA, triple helices, nucleic acid, polypyrimidine, protonated cytosines, major-groove Hoogsteen, Watson-Crick double helix, third strand, binding affinity, electrostatic effects, oligonucleotides, intramolecular triple helices, hairpin, circular dichroism spectroscopy, ultraviolet spectroscopy, Hoogsteen strand, triple helical RNA, deoxyribohomopyrimidine, short duplex DNA, sequence-dependent hybridization reactions, sequence-dependent melting stability, oligomers, thermodynamic parameters, nearest-neighbor, accurate predictions, oligonucleotide-directed, triple-helix formation, pyrimidine-purine-pyrimidine structure, oligonucleotide, triplex, van't Hoff enthalpy, triplets, single-site triplex formation, DNA duplex, thermodynamic parameters, triple helices, pH effects, cytosine, nearest neighbor, electrostatic effects, Hoogsteen strand, enthalpy, entropy, triplex formation, triplex formation, gene expression, nanomaterials, DNA, drug delivery vector, triplex, in vitro, in vivo, single-walled nanotubes, single-strand, physiological conditions, circular dichroism spectroscopy, ultraviolet spectroscopy, DNA melting, atomic force microscopy, electrostatic interaction, biosensing, DNA, polynucleotide structures, three-stranded helices, thermodynamic, single stranded, ionic effects, ligand-polynucleotide interactions, molecular dynamics simulations, DNA, trivalent counterions, divalent counterions, electrostatic interactions, Monte Carlo method, molecular properties, thermodynamic properties, double-helical DNA, oligomers, scanning calorimetry, scanning calorimetry, Monte Carlo simulations, oligonucleotide helices, conformational transitions, thermodynamic effects, nucleic acid, oligomers, polynucleotide, hairpin oligonucleotides, oligoelectrolyte end effect, oligonucleotide dimer helices, dimer oligonucleotides, linkage, allostery, enthalpy, binding scanning calorimetry, tRNA, purine-purine-pyrimidine triple helical, DNAse I, base pair, DNA, purine rich oligonucleotides, triplet, double helices, triple helices, RNA, DNA, pyrimidine strands, purine strands, duplexes, triplexes, third strand, antisense strategy, in vivo, nucleic acid complexes, RNA-DNA, triple helix formation, double helical RNA, RNA-DNA hybrids, triple helical pyrimidine-purine-pyrimidine, binding affinities, pyrimidine, pyrimidine third strand, purine-pyrimidine double helical, oligonucleotides, double helix, sequence-specific binding, purine Watson-Crick, pyr-pur-pyr triple helices, oligonucleotides, DNA, RNA, 2’-O-methyl, single-stranded purine, triple helix formation, Watson-Crick duplexes, purine strand, pH-dependent triplex, triple helical complexes, duplexes, single-stranded, double-stranded, RNA-selective binding, DNA-selective binding, UV-absorption spectrophotometry, molecular modeling, triple helix stability, pyrimidine-purine-pyrimidine motif, three strands, DNA, RNA, third strand, triple helix, polypurine sequence, JUMNA program, polypyrimidine third strand, antiparallel strand, parallel strand, oligoribonucleotide, double helical DNA, DNA oligomers, triple-helical complexes, van't Hoff enthalpy, free energy, cytosine repulsion, nearest neighbor, pyrimidine-purine-pyrimidine triple helices, parallel intramolecular DNA triplex, UV melting curves, nuclear magnetic resonance spectroscopy, duplex, triplex, Hoogsteen pyrimidine strand, Watson-Crick strands, glycosidic torsion angles, nuclear Overhauser effect, NOE, Internucleotide NOEs, molecular dynamics, purine strand, parallel triple helices, thermodynamics, DNA triplex formation, isothermal titration calorimetry, ITC, triplex formation, double-stranded homopurine-homopyrimidine, single-stranded homopyrimidine, calorimetric enthalpy change, binding free energy, van’t Hoff enthalpy, single-stranded DNA, double-stranded DNA, thermodynamic analysis, differential scanning calorimetry, intermolecular DNA triplexes, DNA duplex, van’t Hoff enthalpy, PUR/PYR triplexes, UV melting curves, PUR triplex, triplex, triplet, PYR triplex, purine oligonucleotides, triple helix formation, in vivo,  triple helices, purine strands, pyrimidine strands, ionic condition, pH conditions, triplex formation, third strand, duplex, thermodynamic parameters, Hoogsteen base pairs, Watson-Crick base pairs, oligodeoxynucleotide, hairpin structure, electrophoretic mobility, DNase I, circular dichroism, triple helix formation, Py-Pu-Py DNA triplex, triple-stranded structure, thermodynamic parameters, hydrogen bonding interactions, DNA triplex, DNase I, circular dichroism, triple-stranded DNA, single-stranded, triple formation, Watson-Crick helix, major groove, duplex, Hoogsteen base pair, UV-denaturation, enthalpy, triplex-coil, protonation, C-residues, pyrimidine binding, H-DNA, scanning calorimetric, nuclear magnetic resonance, circular dichroism, duplex, triple helix, base-pair interactions, polymer complex, homopolymer, DNA, thermodynamic parameters, triple-helix formation, sequence-specific, duplex DNA, triplex DNA, RNA, stringency clamping, van't Hoff enthalpy, thermal stability, base mismatch, duplex, triplex formation, triplex helix, single transition, intermolecular triplexes, thymines, pyrimidine strand, thermodynamic parameters, UV melting, purine strand, enthalpy, calorimetry, Monte Carlo simulation, parameter estimation, van’t Hoff plots, calorimetric enthalpies, calorimetry, temperature variation of equilibrium constants, van't Hoff enthalpies, van't Hoff plots, calorimetric enthalpies, thermodynamic models, enthalpy-entropy compensation, heat capacity changes, protein-ligand interactions, computer simulations, binding of nucleotides, ionic strength, tetramer binding, single-stranded oligodeoxyribonucleotides, thermodynamic basis, polypyrimidine DNA, van’ Hoff plots, protein-DNA interaction, hydrophobic effect, heat capacity change, protein-nucleic acid systems, thermodynamic profiles, DNA single-stranded, DNA duplex formation, scanning calorimetry, temperature-dependent, UV absorbance spectroscopy, transition enthalpies, transition entropy, duplex disruption, random-coil, unstacked, intramolecular interactions, nearest-neighbor data, DNA structures, triplexes, tetraplexes, oligodeoxynucleotides, cytidines, tetrameric structure, parallel-stranded duplexes, base pairs, i-motif, cytosine-rich pyrimidine oligodeoxynucleotides, telomeric repeat sequences, intramolecular pairing, thermal denaturation, triplex-forming oligodeoxynucleotides, i-DNA motif, i-motif intramolecular, i-motif intermolecular, triple helix formation, i-motif intramolecular, i-motif intermolecular, i-motif formation, pyrimidine DNA third strands, rich-oligopurine-oligopyrimidine sequences, 2¢-O-methyl, DNA, RNA, i-motif formation, cytosine-rich oligodeoxynucleotides, oligoribonucleotides, RNA pyrimidine oligonucleotides, pyrimidine triplexes, oligonucleotide-directed triple helix formation, purine motif, guanine-rich oligonucleotides, duplex DNA, in vivo, gene inhibition, triple helix formation, guanine-rich oligonucleotides, physiological concentrations, guanine-rich oligonucleotides, guanine quartets, physiological ionic conditions, triple, quadruplex, Ki-ras promoter, triplex DNA, G-rich oligonucleotides, transcription inhibition, RNA polymerase transcription, physiological conditions, intermolecular GA oligonucleotide, intermolecular GT oligonucleotide, triplexes, homoduplex, double-stranded DNA, oligonucleotide self-association, triplex formation, UV spectroscopy, transition enthalpy, homoduplex formation, multivalent cations, monovalent cations, divalent cations, S1 nuclease, helical structures, single-hairpin duplex, low ionic strength, higher ionic strength, double-hairpin tetraplex, ionic strength-dependent equilibrium, single-stranded species, oligomer, random coil, linear duplex, monomer species, dimer species, double-stranded species, hairpin helices, S1 nuclease, UV melting transition, G-tetrads, scanning calorimetry, oligonucleotide, hyperchromism, unstacking, H-bonding, H-bonded tetrads, glycosyl bond, hairpins interact, dimerization regions, antisense oligonucleotides, DNA, RNA, oligonucleotide-based therapeutics, Watson-Crick base pairing, Hoogsteen base pairing, quadruplex structures, triplex-forming oligonucleotides, purine–pyrimidine stretch, triple-helix-forming oligonucleotides, TFO, self-association of the oligonucleotide, parallel duplexes, parallel tetraplexes, duplexes zipper, in vivo"
dato_n = dato.split(", ")

vector = [[l, dato_n.count(l)] for l in set(dato_n)]
dic = dict((l, dato_n.count(l)) for l in set(dato_n))
print(dic)





'''
for i in range(len(dato_n)):
    palabra = dato_n[i]
    contar = dato.count(palabra, i, len(dato))
    dato2 =  dato.replace(palabra, None)
    print("La palabra es: ", palabra)
    print("el numero de veces: ", contar)
'''

#lst = ['Cat', 'Bat', 'Sat', 'Cat', 'Mat', 'Cat', 'Sat']
#print(lst)
#print(dato_n)
#print([[l, lst.count(l)] for l in set(lst)])

#print(dict((l, lst.count(l)) for l in set(lst)))
